modules/tinymce/src/themes/silver/main/ts/ui/statusbar/Statusbar.ts

```diff
@@ -9,6 +9,7 @@ import { SimpleSpec } from '@ephox/alloy';
 
 import Editor from 'tinymce/core/api/Editor';
 import I18n from 'tinymce/core/api/util/I18n';
+import { Logo } from 'tinymce/themes/silver/resources/StatusbarLogo';
```

Comment:
Should this be a relative path since it's something within the silver theme? Normally anything within the `main` dir would need to be relative, but our regular codestyle doesn't really cover this case, so just thought it was worth asking.

Reply:
The problem is that when looking outside of `main/ts` the relative path might change when this source is compiled into the `lib` folder. I figured a named path like this was more consistently reliable.

Reply:
Yeah that's a good point, lets leave it as is then.